charts/kafka-operator/crds/operator-kafka-crd.yaml

```diff
@@ -499,6 +499,47 @@ spec:
                           type: string
                       type: object
                     type: array
+                  envoy:
+                    description: Custom properties for the dedicated envoy deployment
+                      for this broker configuration group.
+                      Applicable only if EnvoyConfig.dedicatedEnvoyPerBrokerGroup is
+                      set to true.
+                    properties:
+                      replicas:
```

Comment:
is there default value?

Reply:
There is no default value (the default is `0` basically). I will add a `minimum: 1` to enforce it if `envoy` property is set.